libdrgn/program.h

```diff
@@ -353,18 +353,18 @@ struct drgn_error *drgn_program_cache_prstatus_entry(struct drgn_program *prog,
 						     uint32_t *ret);
 
 /*
- * Like @ref drgn_program_find_symbol_by_address(), but @p ret is already
- * allocated, we may already know the module, and doesn't return a @ref
- * drgn_error.
+ * Like @ref drgn_program_find_symbol_by_address(), but does not return a lookup
+ * error if the symbol was not found.
```

Comment:
It'd be nice to clarify that it returns `NULL` in `ret` instead if the symbol is not found.